### PR TITLE
test: [M3-8197,M3-8150] - Cypress test for refactored events and placement group

### PR DESCRIPTION
**Pull request by AzureLatte**
## Description 📝
- Clicking "View all events" navigates to Events page at /events.
- Test to confirm flow when selecting existing Placement Group.

## Changes  🔄
- Add assertion to confirm clicking "View all events" navigates to Events page at /events.
- Add a new test to confirm flow when selecting existing Placement Group.

## How to test 🧪
```
yarn cy:run -s "cypress/e2e/core/notificationsAndEvents/events.spec.ts"
yarn cy:run -s "cypress/e2e/core/placementGroups/create-linode-with-placement-groups.spec.ts"
```

## As an Author I have considered 🤔

*Check all that apply*

- [ ] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [ ] ➕ Adding a changeset
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/.changeset/pr-10719-tests-1722001957079.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@linode/manager": Tests
+---
+
+Cypress test for refactored events and placement group ([#10719](https://github.com/linode/manager/pull/10719))
```
Comment (dwiley-akamai) [inline packages/manager/.changeset/pr-10719-tests-1722001957079.md:5]:
```suggestion
E2E coverage for refactored Events and Placement Groups flows ([#10719](https://github.com/linode/manager/pull/10719))
```

File: packages/manager/cypress/e2e/core/placementGroups/create-linode-with-placement-groups.spec.ts [hunk 2]
```diff
@@ -208,4 +212,76 @@ describe('Linode create flow with Placement Group', () => {
       );
     });
   });
+
+  /*
+   * - Confirms UI flow to create a Linode with an existing Placement Group using mock API data.
+   * - Confirms that Placement Group is reflected in create summary section.
+   * - Confirms that outgoing Linode Create API request specifies the selected Placement Group to be attached.
+   */
+  it('can assign existing Placement group during Linode Create flow', () => {
```
Comment (dwiley-akamai) [inline packages/manager/cypress/e2e/core/placementGroups/create-linode-with-placement-groups.spec.ts:221]:
```suggestion
  it('can assign existing Placement Group during Linode Create flow', () => {
```

**Review by dwiley-akamai — APPROVED** [review]
Tests pass locally & in CI ✅

Screenshot 2024-07-30 at 4 54 53 PM

Screenshot 2024-07-30 at 4 55 10 PM

Code review ✅

**Review by cpathipa — APPROVED** [review]
LGTM!

File: packages/manager/.changeset/pr-10719-tests-1722001957079.md (hunk 1 above)
Comment (AzureLatte) [reply packages/manager/.changeset/pr-10719-tests-1722001957079.md:5]: Updated, thank you!

File: packages/manager/cypress/e2e/core/placementGroups/create-linode-with-placement-groups.spec.ts (hunk 2 above)
Comment (AzureLatte) [reply packages/manager/cypress/e2e/core/placementGroups/create-linode-with-placement-groups.spec.ts:221]: Nice catch, thanks!

**Comment by AzureLatte** [comment]
Volumn failure has been fixed in https://github.com/linode/manager/pull/10728

**Pull request merged**